app/src/main/res/layout/dialog_edit_profile_image.xml

```diff
@@ -1,78 +1,84 @@
 <?xml version="1.0" encoding="utf-8"?>
-<LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
+<ScrollView
```

Comment:
Use nested scroll view instead

Reply:
Same for PR https://github.com/fossasia/open-event-attendee-android/pull/2633

Reply:
okay